frontend/src/tests/UserNameInput.spec.ts

```diff
@@ -179,12 +171,83 @@ describe('UserNameInput', () => {
         expect(wrapper.props()).not.toEqual(props);
     });
 
+    it('Should show not found on card when user is not found', async () => {
+        vi.spyOn(
+            AppActlApiService.idirBceidProxyApi,
+            'idirSearch'
+        ).mockImplementation(async () => {
+            return idimIdirSearchMock(false);
+        });
+
+        vi.spyOn(
+            AppActlApiService.idirBceidProxyApi,
+            'bceidSearch'
+        ).mockImplementation(async () => {
+            return idimIdirSearchMock(false);
+        });
+
+
+        // triggers username input change to enable the verify button and click
+        await wrapper.setProps({userId: USER_ID });
+        await verifyButton.trigger('click');
+        await flushPromises();
+
+        // emit setVerifyResult will not be called, when prop domain is I, mock api returns user not found
+        let emitSetVerifyResult = wrapper.emitted('setVerifyResult');
+        expect(emitSetVerifyResult).toBeFalsy();
+
+        const cardEl = wrapper.find('.custom-card').element as HTMLSpanElement;
+        expect(cardEl.textContent).toContain('Username');
+        expect(wrapper.find('#userId').element.textContent).toContain(USER_ID);
+        expect(wrapper.find('#userNotExist').element.textContent).toContain(
+            'User does not exist'
+        );
+
+        // triggers username and domain change to BCeID
+        await wrapper.setProps({ domain: newProps.domain, userId: USER_ID });
```

Comment:
Hi Jota, sorry I forgot that our test file already have the test cases for searching BCeID, and thanks so much for adding more. I wonder if we could rename this newProps to "bceidProps"? and let's create a variable for the "User doest not exist"

Reply:
Hello! Sure can!